dinky-metadata/dinky-metadata-hive/src/test/java/org/dinky/metadata/HiveTest.java

```diff
@@ -76,8 +80,8 @@ public void connectTest() {
         config.setPassword(passwd);
         config.setUrl(url);
         String test = Driver.build(config).test();
-        System.out.println(test);
-        System.err.println("end...");
+        LOGGER.info(test);
+        LOGGER.error("end...");
     }
```

Comment:
Why use `LOGGER.error("...")` ?

Reply:
Oh, This is my fault, because the original log output uses `System.err.println`. Maybe this is the wrong use, but I directly converted to `LOGGER.error`. Later, I will modify the log level according to the actual output